docs/README.md

```diff
@@ -9,6 +9,8 @@
 
 ---
 
+Try out our example notebooks here: https://colab.research.google.com/github/lf1-io/padl/
+
```

Comment:
Can you update this README with your changes as well?
```python
### Try out PADL in Colab notebooks
1. [MNIST](https://colab.research.google.com/github/lf1-io/padl/blob/main/notebooks/01_MNIST_using_padl.ipynb#scrollTo=bd560eb8)
2. [Simple NLP example](https://colab.research.google.com/github/lf1-io/padl/blob/main/notebooks/02_nlp_example.ipynb)
3. [Sentiment Analysis - NLP](https://colab.research.google.com/github/lf1-io/padl/blob/main/notebooks/03_Sentiment_Analysis_with_padl.ipynb)
```

Reply:
what do you mean by that?

Reply:
Add your changes to this README. It's in the `docs/` folder. It is not auto built by the `make docs`.

Reply:
This is the README that appears on our docs https://lf1-io.github.io/padl/ page

Reply:
i have updated this.